src/compas_rhino/artists/robotmodelartist.py

```diff
@@ -197,6 +200,7 @@ def _add_mesh_to_doc(self, mesh):
                 attr.Name = name
 
             obj.CommitChanges()
+        return guid
```

Comment:
It looks like most other compas_rhino `draw` methods return a list of guids even if there's only one (see MeshArtist).  So I think this should be:
```suggestion
        return [guid]
```